AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Core/Conversion.cs

```diff
@@ -26,74 +27,75 @@ public ConversionResponse ConvertAzurePipelineToGitHubAction(string input)
 
             //Triggers and variables are hard, as there are two data types for each that can exist, so we need to go with the most common type and handle the less common type with exceptions. 
             //There are 4 combinations here, simple/simple, simple/complex, complex/simple, and complex/complex
```

Comment:
@samsmithnz I have refactored this method to make it more readable.

I would strongly suggest using var when it's clear what the object type is, as in the below example variableList is clearly a List<string>;
```
List<string> variableList = new List<string>();
```
vs.
```
var variableList = new List<string>();
```
The local field name+class names(+generics) are very long, so it becomes a case of 'not being able to see the wood for the trees' i.e. visual code overload of complexity.

Reply:
I like what you've done with the success branch. 

Not so much the var. I appreciate the comment about readability, that is fair. For me it's a personal preference to have strong types. I don't care enough to fail the review, but enough to write it here. ;) :)